### PR TITLE
Improve Vtiger-odoo12

### PR DESCRIPTION
**Pull request by dhiren-scs**
Fix the multiple line issue of saleorder and Purchase order.

### CODE REVIEW

File: vtiger_connector_purchase/models/res_company.py [hunk 1]
```diff
@@ -8,6 +8,8 @@
 from datetime import datetime
 from urllib.request import urlopen, Request
 from urllib.parse import urlencode
+import requests
+from requests.auth import HTTPBasicAuth
```
Comment (hiren-serpentcs) [inline vtiger_connector_purchase/models/res_company.py]: Remove HTTPBasicAuth if we are not using it anywhere

File: vtiger_connector_purchase/models/res_company.py [hunk 2]
```diff
@@ -59,51 +61,60 @@ def sync_vtiger_purchase_order(self):
             if result.get('success'):
                 self.update_existing_order(result)
                 for res in result.get('result', []):
+                    po_id = res.get('id')
+                    line_url = company.vtiger_server
+                    line_url += "/restapi/v1/vtiger/default/retrieve?id=" + po_id
```
Comment (hiren-serpentcs) [inline vtiger_connector_purchase/models/res_company.py:69]: Add docstring for the second api call

**Review by hiren-serpentcs — CHANGES_REQUESTED** [review]

File: vtiger_connector_sales/models/res_company.py [hunk 3]
```diff
@@ -157,33 +169,40 @@ def sync_vtiger_sale_Quotes(self):
             if result.get('success'):
                 self.update_existing_sale_Quotes(result)
```
Comment (hiren-serpentcs) [inline vtiger_connector_sales/models/res_company.py]: check update_existing_sale_order and update_existing_sale_Quotes methods, they both are having exact same code

**Review by hiren-serpentcs — CHANGES_REQUESTED** [review]

**Comment by JayVora-SerpentCS** [comment]
@hiren-serpentcs do tell me when its ready to go

**Review by hiren-serpentcs — APPROVED** [review]

**Comment by hiren-serpentcs** [comment]
@JayVora-SerpentCS 
LGTM :thumbsup:

Do we need to assign this to QA?

**Comment by JayVora-SerpentCS** [comment]
Talk to Savitri. She would assign to new comer Keval.
I am merging meanwhile.
Thanks.

**Pull request merged**